code/modules/nano/nanomanager.dm

```diff
@@ -7,6 +7,41 @@
 	var/list/processing_uis = list()
 	// a list of asset filenames which are to be sent to the client on user logon
 	var/list/asset_files = list()
+	// a list of asset files that get added to the RSC directly so as not to lag client joins.
+	var/list/rsc_files = list(
+		'nano/css/codemirror.css',
+		'nano/css/icons.css',
+		'nano/css/layout_basic.css',
+		'nano/css/layout_default.css',
+		'nano/css/lesser-dark.css',
+		'nano/css/shared.css',
```

Comment:
i don't think users would appreciate hilariously broken UI's either because it's fucking impossible to resolve issues

but whatever nano's already a piece of shit to debug, just another thing in the way